core/src/test/java/org/infinispan/xsite/offline/AsyncTimeBasedOfflineTest.java

```diff
@@ -98,7 +98,11 @@ private void assertOnline(String cacheName, int index, String targetSiteName) {
    private void assertEventuallyOffline(String cacheName, int index) {
       OfflineStatus status = backupSender(cacheName, index).getOfflineStatus(SFO);
       assertTrue(status.isEnabled());
-      eventually(() -> "Site " + SFO + " is online. status=" + status, status::isOffline);
+      eventually(() -> "Site " + SFO + " is online. status=" + status, () -> {
+         //since ISPN-10848, the status only changes after an ACK
```

Comment:
I'd rather update the test to send the ACK and wait for that instead of updating the status manually.

Although TBH I read the full test and I couldn't figure out what could cause the SFO site to go offline (or online for that matter, because a comment in `createSites()` says SFO is offline).

Reply:
the ack is sent when you invoke the `cache.put()` and it will start the timer. There is no need for another ack. 

> Although TBH I read the full test and I couldn't figure out what could cause the SFO site to go offline (or online for that matter, because a comment in `createSites()` says SFO is offline).

SFO site is never created (to simulate an offline site). When a cache starts, it assumes all sites are online until it starts to send requests to it (in this case, a `put()`)

Reply:
> the ack is sent when you invoke the `cache.put()` and it will start the timer. There is no need for another ack.

True, but that's only because you update the status manually, something that would never happen in production.

> 
> SFO site is never created (to simulate an offline site). When a cache starts, it assumes all sites are online until it starts to send requests to it (in this case, a `put()`)

Ok, but I think you're overloading "offline" here to mean something different than what it means in production code. If the cache assumes SFO is online, then it's online, because a site can only be online or offline from the perspective of the cache that's trying to send backup commands to it.